src/PKSim.Core/Services/SimulationConfigurationTask.cs

```diff
@@ -107,15 +108,20 @@ public SimulationConfiguration CreateFor(Simulation simulation, bool shouldValid
          //STEP8: Add Observers
          module.Add(_modelObserverQuery.AllObserversFor(module.Molecules, simulation));
 
-         //STEP9 once all building blocks have been created, we need to create the default parameter and molecule values values 
-         var initialConditions = _initialConditionsCreator.CreateFor(module, simulation);
-         module.Add(initialConditions);
+         //STEP9 once all building blocks have been created, we need to create the default parameter and compound molecule values
+         var allDefinedProteins = module.Molecules.Where(x => simulation.Individual.AllDefinedMolecules().AllNames().Contains(x.Name)).ToList();
+         var allOtherMolecules = module.Molecules.Except(allDefinedProteins).ToList();
```

Comment:
What about 2 PORES Molecules here .
@rwmcintosh Can you create a model using TWO PORES and make sure it behaves the same as in v11?